elm/utilities/parse.py

```diff
@@ -13,7 +13,7 @@
 logger = logging.getLogger(__name__)
 
 
-def is_multi_col(text, separator="    "):
+def is_multi_col(text, separator="    ", threshold_ratio=0.35):
```

Comment:
Why implement this in addition to `PDFtoTXT.is_double_col()`? Why not remove `is_double_col` from PDFtoTXT or have is_multi_col be a static method in PDFtoTXT? (latter is probably prefered).

Reply:
This was mirroring the previous set up where `PDFtoTXT.is_double_col()` uses `is_multi_col()` but if there is no reason to preserve that structure I can set up the static method.

Reply:
oh yeah and `is_multi_col` is in a utilities module.... ehhhh i guess it doesnt really matter haha

Reply:
I think you should remove is_double_col from PDFtoTXT and just call the utility function where required without it being attached to PDFtoTXT

Reply:
Sounds good, will do!